publish/samplesite/scripts/controls/sidePane/eventViewer/EventViewPane.tsx

```diff
@@ -147,6 +148,10 @@ export default class EventViewPane extends React.Component<
                         ))}
                     </span>
                 );
+            case PluginEventType.PendingFormatStateChanged:
+                const formatState = event.formatState;
+                const keys = Object.keys(formatState) as (keyof PendableFormatState)[];
+                return <span>{keys.map(key => `${key}=${event.formatState[key]}; `)}</span>;
```

Comment:
Maybe I'm confused by this syntax, but does this mean a single span could have multiple keys?

Reply:
yes, it shows multiple keys in one span. This is for the sample site only.